src/game/client/components/menus_browser.cpp

```diff
@@ -36,7 +36,7 @@ static ColorRGBA PlayerBackgroundColor(bool Friend, bool Clan, bool Afk, bool In
 		i = 1;
 	else
 		i = 2;
-	return (Afk ? COLORS_AFK[i] : COLORS[i]).WithAlpha(Inside ? 0.45f : 0.3f);
+	return (Afk ? COLORS_AFK[i] : COLORS[i]).WithAlpha((Inside ? 0.45f : 0.3f) + InSelectedServer * 0.12);
```

Comment:
Multiplying a bool with a float/double seems weird.

```suggestion
	return (Afk ? COLORS_AFK[i] : COLORS[i]).WithAlpha(0.3f + (Inside ? 0.15f : 0.0f) + (InSelectedServer ? 0.12f : 0.0f));
```

Reply:
what is weird about it? it gets promoted.